backend/tests/integration/records/serializers/json/test_literature.py

```diff
@@ -900,3 +900,31 @@ def test_record_link_to_hepdata_in_detail_view(inspire_app):
         response.json["metadata"]["external_system_identifiers"]
         == expected_response_metadata
     )
+
+
+def test_literature_detail_links(inspire_app):
+    expected_status_code = 200
+    record = create_record("lit")
+    cn = record["control_number"]
+    expected_links = {
+        "bibtex": f"http://localhost:5000/literature/{cn}?format=bibtex",
+        "citations": f"http://localhost:5000/literature/{cn}/citations",
```

Comment:
I thought we wanted to redirect to a search with `q=refersto:recid:{cn}` here, as the `citations` endpoint is a specific serialization for the frontend (that might be removed soon).

Reply:
This was not mentioned in requirements nor in Tech notes.

Reply:
Actually in requirements this specific link was mentioned as example for citations `"https://inspirehep.net/api/literature/451647/citations?page=1&size=25"`

Reply:
And from what I can see this endpoint will stay there for now. Only UI won't call it anymore.

Reply:
But that endpoint returns only the UI data, not a full record. I think that's confusing. IIRC, we discussed this with @drjova and decided to use the search for this. WDYT @drjova ?

Reply:
Why stuff like this is always appearing during review... Here it won't be too difficult to change it but still it's wrong. If requirements was accepted already, and refinement for the task was done and task was almost finished and it's now on review, we shouldn't change requirements but bugfix task should be created for this.

Reply:
@pazembrz this is totally not your fault and I know it's frustrating. It's good that we cautch such things during review and we have to be more careful with the requirements in the future. This is my fault, @michamos you're right we dicided to go for the search results and I didn't update the US afterwards.

Reply:
I have weird feeling that you are trying to calm me down. Was I aggressive in the previous post? If I was I'm sorry. It wasn't my intention. I was trying to say that our workflow is weird and we shouldn't look who's fault was that, but on the moment when we see bug/problem like that a new bug should be created in Jira. It could be fixed by me, even in same PR but in separate commit so we could keep cohesion (is this the right word?) between Jira tasks and our code. So in the future we could understand what happened here (why task requirements are different from code) without looking for closed PR and reading all comments in it.

Reply:
In that case is a problem that can be solved immidietly in the review, which is a good. I'll update the US with this information.

Reply:
:+1: